### PR TITLE
Atlas Ship

### PR DESCRIPTION
**Pull request by Emisse**
Atlas is a low population ship based off the goonstation map of the same name. All credit for original layout of the ship go to Gannets. This map has been modified for ss14 and changes were made in various places although the overall layout should feel similar. Passive ship power lasts around 10 minutes. This ship runs off AME/solar and can sustain off 1 core AME for around 30 minutes. Dual solars sustains for a bit short, and combined can be net positive power return. Atmos is a 2 tank setup for bare minimum performance due to ship size. The tanks are on board the ship so I have included an 'oh shit' button in case a window to a miner is broken open which will expose the tanks to vacuum. This station is fully functional as a ship however I did not include gyroscopes or a shuttle console to start to prevent salvage roundstart CBT. These will have to be built and assembled to make it functional. 
you know the drill heres some candid screenshots 
bridge
image
medbay
image
hydroponics
image
science
image
south hangar bay
image
engineering
image
thank you for reading and hope you enjoy

:cl: Emisse
- add: Atlas, a low-pop ship map adapted from Goonstation (SS13). Major credit goes to all those who worked on the original layout of Atlas on Goonstation.

### CODE REVIEW

File: Resources/Prototypes/Maps/atlas.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+- type: gameMap
+  id: atlasship
+  mapName: 'Atlas Ship'
+  mapNameTemplate: '{0} Atlas Ship {1}'
+  nameGenerator:
+    !type:NanotrasenNameGenerator
+    prefixCreator: '14'
```
Comment (moonheart08) [inline Resources/Prototypes/Maps/atlas.yml:7]:
```suggestion
    prefixCreator: 'R4' # R4407/Goon. GS isn't as cool sounding.
```

File: Resources/Prototypes/Maps/atlas.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+- type: gameMap
+  id: atlasship
+  mapName: 'Atlas Ship'
+  mapNameTemplate: '{0} Atlas Ship {1}'
```
Comment (moonheart08) [inline Resources/Prototypes/Maps/atlas.yml:4]:
```suggestion
  id: atlas
  mapName: 'Atlas'
  mapNameTemplate: '{0} Atlas {1}'
```

**Comment by moonheart08** [comment]
atlasship-0

**Comment by mirrorcult** [comment]
new development env map discovered

**Comment by Emisse** [comment]
i wish renderer would do the decals that arent tiled snapped properly, all the plants look like theyre spilling out lol

**Comment by moonheart08** [comment]
image
Could shift this a bit to not be a 1-wide hall.

**Comment by Emisse** [comment]
ill write that down for later i been mapping all day and need a break lol

**Comment by moonheart08** [comment]
> ill write that down for later i been mapping all day and need a break lol

No problem. Do you want it merged now for testing, it otherwise lgtm and would let us get issue reports.

**Comment by Emisse** [comment]
Yea that would be nice but it'll probably only be testable during low pop hours due to size.

**Comment by moonheart08** [comment]
> Yea that would be nice but it'll probably only be testable during low pop hours due to size.

Bet. (we can just run two)

**Comment by Gannets** [comment]
Hey it's that map I made.

**Comment by PaulRitter** [comment]
giving the render a quick look. am i mistaken or are the north dockingports the wrong way around?

**Comment by Pangogie** [comment]
> giving the render a quick look. am i mistaken or are the north dockingports the wrong way around?

They're fine ingame. Map renderer doesn't like rotations.

**Pull request merged**